Godeps/Godeps.json

```diff
@@ -2791,6 +2791,58 @@
 			"Rev": "c2ac40f1adf8c42a79badddb2a2acd673cae3bcb"
 		},
 		{
+			"ImportPath": "k8s.io/metrics/pkg/apis/custom_metrics",
```

Comment:
I think I'd probably be in favor of moving k8s.io/metrics to staging.  In the short term, it'll also make regenerating the types easier (although I'll need to port a couple of my generator changes, which add pretty simple new tags).